infrastructure/versions.tf

```diff
@@ -1,3 +1,9 @@
 terraform {
   required_version = ">= 0.13"
+  required_providers {
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "= 1.6.0"
```

Comment:
haven't you changed the postgres module to require 2.0.0 ?

and please don't put versions in a different place to the rest of them, either move this back to state, or move the others here